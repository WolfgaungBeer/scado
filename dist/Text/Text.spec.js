'use strict';

var _react = require('react');

var _react2 = _interopRequireDefault(_react);

var _reactTestRenderer = require('react-test-renderer');

var _reactTestRenderer2 = _interopRequireDefault(_reactTestRenderer);

var _Wrapper = require('../../__mocks__/Wrapper');

var _Wrapper2 = _interopRequireDefault(_Wrapper);

var _ = require('./');

var _2 = _interopRequireDefault(_);

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }

it('renders the Text Component correctly', function () {
    var tree1 = _reactTestRenderer2.default.create(_react2.default.createElement(
        _Wrapper2.default,
        null,
        _react2.default.createElement(
            _2.default.H1,
            null,
            'Das ist ein Text'
        )
    )).toJSON();
    expect(tree1).toMatchSnapshot();

    var tree2 = _reactTestRenderer2.default.create(_react2.default.createElement(
        _Wrapper2.default,
        null,
        _react2.default.createElement(
            _2.default.H2,
            null,
            'Das ist ein Text'
        )
    )).toJSON();
    expect(tree2).toMatchSnapshot();

    var tree3 = _reactTestRenderer2.default.create(_react2.default.createElement(
        _Wrapper2.default,
        null,
        _react2.default.createElement(
            _2.default.H3,
            null,
            'Das ist ein Text'
        )
    )).toJSON();
    expect(tree3).toMatchSnapshot();

    var tree4 = _reactTestRenderer2.default.create(_react2.default.createElement(
        _Wrapper2.default,
        null,
        _react2.default.createElement(
            _2.default.H4,
            null,
            'Das ist ein Text'
        )
    )).toJSON();
    expect(tree4).toMatchSnapshot();

    var tree5 = _reactTestRenderer2.default.create(_react2.default.createElement(
        _Wrapper2.default,
        null,
        _react2.default.createElement(
            _2.default.H5,
            null,
            'Das ist ein Text'
        )
    )).toJSON();
    expect(tree5).toMatchSnapshot();

    var tree6 = _reactTestRenderer2.default.create(_react2.default.createElement(
        _Wrapper2.default,
        null,
        _react2.default.createElement(
            _2.default.H6,
            null,
            'Das ist ein Text'
        )
    )).toJSON();
    expect(tree6).toMatchSnapshot();

    var tree7 = _reactTestRenderer2.default.create(_react2.default.createElement(
        _Wrapper2.default,
        null,
        _react2.default.createElement(
            _2.default.P,
            null,
            'Das ist ein Text'
        )
    )).toJSON();
    expect(tree7).toMatchSnapshot();

    var tree8 = _reactTestRenderer2.default.create(_react2.default.createElement(
        _Wrapper2.default,
        null,
        _react2.default.createElement(
            _2.default.A,
            null,
            'Das ist ein Text'
        )
    )).toJSON();
    expect(tree8).toMatchSnapshot();

    var tree9 = _reactTestRenderer2.default.create(_react2.default.createElement(
        _Wrapper2.default,
        null,
        _react2.default.createElement(
            _2.default.Span,
            null,
            'Das ist ein Text'
        )
    )).toJSON();
    expect(tree9).toMatchSnapshot();

    var tree10 = _reactTestRenderer2.default.create(_react2.default.createElement(
        _Wrapper2.default,
        null,
        _react2.default.createElement(
            _2.default.Label,
            null,
            'Das ist ein Text'
        )
    )).toJSON();
    expect(tree10).toMatchSnapshot();

    var tree11 = _reactTestRenderer2.default.create(_react2.default.createElement(
        _Wrapper2.default,
        null,
        _react2.default.createElement(
            _2.default.Span,
            { color: 'error', scale: 'xl' },
            'Das ist ein Text'
        )
    )).toJSON();
    expect(tree11).toMatchSnapshot();
});