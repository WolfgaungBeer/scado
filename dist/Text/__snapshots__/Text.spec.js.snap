// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the Text Component correctly 1`] = `
<h1
  className="sc-jKJlTe ldvWdm"
>
  Das ist ein Text
</h1>
`;

exports[`renders the Text Component correctly 2`] = `
<h2
  className="sc-eNQAEJ jUxSlh"
>
  Das ist ein Text
</h2>
`;

exports[`renders the Text Component correctly 3`] = `
<h3
  className="sc-hMqMXs ddVDsl"
>
  Das ist ein Text
</h3>
`;

exports[`renders the Text Component correctly 4`] = `
<h4
  className="sc-kEYyzF fqBdfp"
>
  Das ist ein Text
</h4>
`;

exports[`renders the Text Component correctly 5`] = `
<h5
  className="sc-kkGfuU nFGvk"
>
  Das ist ein Text
</h5>
`;

exports[`renders the Text Component correctly 6`] = `
<h6
  className="sc-iAyFgw gyTSdg"
>
  Das ist ein Text
</h6>
`;

exports[`renders the Text Component correctly 7`] = `
<p
  className="sc-hSdWYo zMyhk"
>
  Das ist ein Text
</p>
`;

exports[`renders the Text Component correctly 8`] = `
<a
  className="sc-eHgmQL jIKWEj"
>
  Das ist ein Text
</a>
`;

exports[`renders the Text Component correctly 9`] = `
<span
  className="sc-jWBwVP jbbQAr"
>
  Das ist ein Text
</span>
`;

exports[`renders the Text Component correctly 10`] = `
<label
  className="sc-cvbbAY iyxLWy"
>
  Das ist ein Text
</label>
`;

exports[`renders the Text Component correctly 11`] = `
<span
  className="sc-jWBwVP khyscU"
  color="error"
  scale="xl"
>
  Das ist ein Text
</span>
`;
